src/main/resources/edu/cornell/kfs/sys/cu-spring-sys.xml

```diff
@@ -269,6 +269,7 @@
 	<bean id="preferencesDao" parent="preferencesDao-parentBean"/>
 	<bean id="preferencesDao-parentBean" abstract="true" class="${kfs.documentstore.class}">
 		<property name="documentStoreClient" ref="documentStoreClient" />
+		<property name="tableNamePrefix" value="${kfs.documentstore.tablename.prefix}" />
```

Comment:
This bean setup assumes that the DAO implementation has a tableNamePrefix setter. However, this changeset is only adding the setter to the Dynamo implementation, so this configuration will not work for our custom Mongo implementation bean. Perhaps the setter should be added to our Mongo implementation as well; or, perhaps a sub-interface of PreferencesDao could be used to require the setter or to at least provide a Java 8 "default" implementation of it.